src/inverseRobot.js

```diff
@@ -24,7 +24,32 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  let newRobot = {};
+  const values = Object.keys(robot);
+  const keys = Object.values(robot);
+
+  for (let x = 0; x < keys.length - 1; x++) {
+    let j = 1;
+
+    if (x > 0) {
+      j += x;
+    }
+
+    for (j; j < keys.length; j++) {
+      if (keys[x] !== keys[j]) {
+        continue;
+      }
+      newRobot = null;
+
+      return newRobot;
+    }
+  }
+
+  for (let i = 0; i < keys.length; i++) {
```

Comment:
```suggestion
  for (let i = 0; i < keys.length; i++) {
    if (newRobot.hawOwnProperty(keys[i])) {
      return null;
    }
```
якось так, думаю логіка зрозіміла